src/win32/InstallerScripts.vbs

```diff
@@ -422,3 +422,32 @@ Private Function GetUserSID()
         GetUserSID = oWMI.SID
     End If
 End Function
+
+Public Function VerifyOrCreateRegistryKey()
+    On Error Resume Next
+    Dim strKeyPath, oReg
+    Dim objCtx, objLocator, objServices
+    Const HKEY_LOCAL_MACHINE = &H80000002
+
+    Set objCtx = CreateObject("WbemScripting.SWbemNamedValueSet")
+    objCtx.Add "__ProviderArchitecture", 64
+    objCtx.Add "__RequiredArchitecture", True
+
+    Set objLocator = CreateObject("WbemScripting.SWbemLocator")
+    Set objServices = objLocator.ConnectServer(".", "root\default", "", "", , , , objCtx)
+    Set oReg = objServices.Get("StdRegProv")
+
+    strKeyPath = "SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\wazuh-agent.exe"
+
+    oReg.CreateKey HKEY_LOCAL_MACHINE, strKeyPath
+    oReg.SetStringValue HKEY_LOCAL_MACHINE, strKeyPath, "DumpFolder", "C:\Users\wazuh\AppData\Local\CrashDumps"
```

Comment:
The user must be a variable (even the whole path) since it will probably not be wazuh, and it can be even on a different drive

Reply:
Change the path to `%LOCALAPPDATA%\WazuhCrashDumps`.